src/components/container/AppDownload/AppDownload.tsx

```diff
@@ -0,0 +1,66 @@
+﻿import React, { useState } from 'react'
+import './AppDownload.css'
+import MyDataHelps, { ProjectInfo } from '@careevolution/mydatahelps-js'
+import '@fortawesome/fontawesome-svg-core/styles.css';
+import PlatformSpecificContent from '../PlatformSpecificContent/PlatformSpecificContent';
+import { useInitializeView } from '../../../helpers/Initialization';
+import Card from '../../presentational/Card';
+import TextBlock from '../../presentational/TextBlock';
+import googlePlayDownload from '../../../assets/google-play-download.svg';
+import appStoreDownload from '../../../assets/app-store-download.svg';
+import language from "../../../helpers/language";
+import LocalizedString from "../../presentational/LocalizedString";
+
+export interface AppDownloadProps {
+	previewProjectPlatforms?: string[]
```

Comment:
maybe some history I am missing or something, but is there a reason for the "preview" prefix?

Reply:
Yes.  It is because those properties are only used to configure the "preview" rendering of this component (and sub components).  Those values are not actually configuration for this component.  They just configure the fake state that is to be used when "previewing" the component in the storybook or during testing.